app/views/provider_interface/offers/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= render ProviderInterface::ApplicationChoiceHeaderComponent.new(application_choice: @application_choice, provider_can_respond: provider_user_can_make_decisions) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-heading-l govuk-!-margin-bottom govuk-!-font-size-36">Offer</div>
+      <% if provider_user_can_make_decisions %>
+        <p class="govuk-body govuk-!-margin-bottom-7">
+          <% if %i[pending_conditions recruited].include?(@application_choice.status.to_sym) %>
+            <%= govuk_link_to 'Defer offer', provider_interface_application_choice_new_defer_offer_path(@application_choice) %>
+          <% end %>
+
+          <% if %i[offer recruited].include?(@application_choice.status.to_sym) %>
+            <%= govuk_link_to 'Withdraw offer', provider_interface_application_choice_withdraw_offer_path(@application_choice) %>
```

Comment:
we might need a bit of spacing when both links are displayed when an application is recruited

Reply:
After amending the conditions for displaying the link, defer and withdraw offer should now never appear at the same time so spacing isn't required.